data/oot/pool.csv

```diff
@@ -11,7 +11,7 @@ Kokiri Forest GS Night Adult,         gs,             KOKIRI_FOREST,
 Malon Egg,                            npc,            HYRULE_CASTLE,            MALON_EGG,                  CHICKEN
 Zelda's Letter,                       npc,            CASTLE_COURTYARD,         ZELDA_LETTER,               ZELDA_LETTER
 Zelda's Song,                         npc,            CASTLE_COURTYARD,         ZELDA_SONG,                 SONG_ZELDA
-Lost Woods Bridge,                    npc,            LOST_WOODS,               SARIA_OCARINA,              OCARINA
+LW Gift from Saria,                   npc,            LOST_WOODS,               SARIA_OCARINA,              OCARINA
```

Comment:
Please spell "Lost Woods" for consistency with other checks

Reply:
i did this for the eventual parity with OOTR

Reply:
We have different conventions for consistency between OoT checks and MM checks

Reply:
i do suppose GV could mean Gerudo Valley or Goron Village

Reply:
oh whoops, forgot to rename it in the yml